api/tests.py

```
@@ -1,3 +1,38 @@
-# Create your tests here.
+from django.contrib.auth import get_user_model
+from django.test import Client, TestCase
+from django.utils import timezone
+
+from .models import Item
+
+
+class ItemsCollectionViewTests(TestCase):
+	def setUp(self) -> None:
+		self.client = Client()
+		self.owner = get_user_model().objects.create_user(username="owner", password="pass1234")
```

Ruff (0.14.13)

Possible hardcoded password assigned to argument: "password"

(S106)